scavenger-agent-java/src/test/java/com/navercorp/scavenger/javaagent/scheduling/SchedulerTest.java

```diff
@@ -81,7 +88,9 @@ public class SchedulerTest {
     public void setUp() throws Exception {
         lenient().when(codeBaseScannerMock.scan())
             .thenReturn(new CodeBase(List.of(Method.createTestMethod()), "fingerprint"));
-        sut = new Scheduler(invocationRegistry, new Config(new Properties()), publisher, codeBaseScannerMock);
+        Properties properties = new Properties();
+        properties.setProperty("callStackTraceMode", "true");
```

Comment:
To make it more clear, it also needs to be tested when `callStackTraceMode` is `false`